app/react/src/client/preview/render.tsx

```diff
@@ -32,10 +32,10 @@ class ErrorBoundary extends React.Component<{
     }
   }
 
-  componentDidCatch({ message, stack }: Error) {
-    const { showError } = this.props;
+  componentDidCatch(err: Error) {
+    const { showException } = this.props;
     // message partially duplicates stack, strip it
```

Comment:
Let's actually do it again

Reply:
Hmm will wait for @tmeasday 's feedback on this. Apparently it interacts with Chromatic in some way

Reply:
chromatic can't test pages with errors anyway

Reply:
Please at least remove the comment if we're not actually stripping anything

Reply:
@Hypnosphi FYI the issue here was that tools like Chromatic listen to to the `storyThrewException` event in order to show the user a stack trace of broken stories. This was reverting a regression where the wrong event was fired in that case.

Would be good to handle the distinction between "user code error" and "react error" somehow though.